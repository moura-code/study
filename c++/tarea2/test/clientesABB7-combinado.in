crearClientesABB
liberarClientesABB
crearClientesABB
crearCliente 10 Brunilda Gomez 34
agregarAClientesABB
crearCliente 20 Casimira Ruiz 22
agregarAClientesABB
crearCliente 16 Zenobia Garcia 40
agregarAClientesABB
crearCliente 24 Brunilda Perez 16
agregarAClientesABB
crearCliente 30 Zenobia Gonzalez 22
agregarAClientesABB
crearCliente 31 Rufino Fernandez 24
agregarAClientesABB
crearCliente 32 Casimira Perez 16
agregarAClientesABB
crearCliente 40 Ubaldo Garcia 21
agregarAClientesABB
crearCliente 8 Esmeralda Fernandez 23
agregarAClientesABB
crearCliente 4 Filemon Lopez 42
agregarAClientesABB
crearCliente 6 Casimira Martinez 18
agregarAClientesABB
crearCliente 7 Rufino Martinez 25
agregarAClientesABB
alturaClientesABB
existeEnClientesABB 6
existeEnClientesABB 31
existeEnClientesABB 2
existeEnClientesABB 18
maxIdClientesABB
edadPromedioClientesABB
removerDeClientesABB 30
removerDeClientesABB 8
removerDeClientesABB 31
removerDeClientesABB 24
alturaClientesABB
existeEnClientesABB 6
existeEnClientesABB 31
maxIdClientesABB
edadPromedioClientesABB
liberarClientesABB
Fin
