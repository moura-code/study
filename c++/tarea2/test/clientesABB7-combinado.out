Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> El clientesABB ha sido creado de forma exitosa.
2> ClientesABB liberado con exito.
3> El clientesABB ha sido creado de forma exitosa.
4> Cliente creado de forma exitosa.
5> Se ha agregado el cliente al clientesABB de forma exitosa.
6> Cliente creado de forma exitosa.
7> Se ha agregado el cliente al clientesABB de forma exitosa.
8> Cliente creado de forma exitosa.
9> Se ha agregado el cliente al clientesABB de forma exitosa.
10> Cliente creado de forma exitosa.
11> Se ha agregado el cliente al clientesABB de forma exitosa.
12> Cliente creado de forma exitosa.
13> Se ha agregado el cliente al clientesABB de forma exitosa.
14> Cliente creado de forma exitosa.
15> Se ha agregado el cliente al clientesABB de forma exitosa.
16> Cliente creado de forma exitosa.
17> Se ha agregado el cliente al clientesABB de forma exitosa.
18> Cliente creado de forma exitosa.
19> Se ha agregado el cliente al clientesABB de forma exitosa.
20> Cliente creado de forma exitosa.
21> Se ha agregado el cliente al clientesABB de forma exitosa.
22> Cliente creado de forma exitosa.
23> Se ha agregado el cliente al clientesABB de forma exitosa.
24> Cliente creado de forma exitosa.
25> Se ha agregado el cliente al clientesABB de forma exitosa.
26> Cliente creado de forma exitosa.
27> Se ha agregado el cliente al clientesABB de forma exitosa.
28> La altura del clientesABB es 7.
29> El cliente con id 6 pertenece al clientesABB.
30> El cliente con id 31 pertenece al clientesABB.
31> El cliente con id 2 NO pertenece al clientesABB.
32> El cliente con id 18 NO pertenece al clientesABB.
33> El mayor id en el clientesABB es 40.
34> La edad promedio del clientesABB es 25.25.
35> El cliente con id 30 se removio del clientesABB.
36> El cliente con id 8 se removio del clientesABB.
37> El cliente con id 31 se removio del clientesABB.
38> El cliente con id 24 se removio del clientesABB.
39> La altura del clientesABB es 4.
40> El cliente con id 6 pertenece al clientesABB.
41> El cliente con id 31 NO pertenece al clientesABB.
42> El mayor id en el clientesABB es 40.
43> La edad promedio del clientesABB es 27.25.
44> ClientesABB liberado con exito.
45> Fin.
